Enable distro_bootcmd support (doc/README.distro) for omap5 targets.

Index: u-boot/include/configs/ti_omap5_common.h
===================================================================
--- u-boot.orig/include/configs/ti_omap5_common.h
+++ u-boot/include/configs/ti_omap5_common.h
@@ -58,6 +58,21 @@
 #include <environment/ti/mmc.h>
 #include <environment/ti/nand.h>
 
+#define BOOT_TARGET_DEVICES(func) \
+	func(MMC, mmc, 0) \
+	func(MMC, mmc, 1) \
+	func(PXE, pxe, na) \
+	func(DHCP, dhcp, na)
+
+#ifdef CONFIG_BOOTCOMMAND
+#undef CONFIG_BOOTCOMMAND
+#endif
+#define CONFIG_BOOTCOMMAND \
+	"run findfdt; " \
+	"run distro_bootcmd"
+
+#include <config_distro_bootcmd.h>
+
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	DEFAULT_LINUX_BOOT_ENV \
 	DEFAULT_MMC_TI_ARGS \
@@ -67,6 +82,7 @@
 	DFUARGS \
 	NETARGS \
 	NANDARGS \
+	BOOTENV \
 
 /*
  * SPL related defines.  The Public RAM memory map the ROM defines the
